Makefile

```diff
@@ -13,7 +13,7 @@ LDFLAGS += -X "github.com/pingcap/pd/server.PDGitHash=$(shell git rev-parse HEAD
 # See more: https://godoc.org/path/filepath#Match
 COVERIGNORE := "cmd/pd-server/main.go,server/api/bindata_assetfs.go"
 
-default: build
+default: build ctl
```

Comment:
put  `ctl` in `build` directly. 

`make build` can build `pd-server` and `pd-ctl`.